src/v/datalake/coordinator/tests/CMakeLists.txt

```diff
@@ -25,7 +25,7 @@ rp_test(
 )
 
 rp_test(
-  UNIT_TEST
+  FIXTURE_TEST
```

Comment:
While you're here, can you also update the tests in datalake/tests? They also appear to run into this